source/conf.py

```diff
@@ -49,10 +57,15 @@
     'q2doc.qiime1',
     'q2doc.checkpoint',
     'q2doc.command_block',
-    'q2doc.plugin_directory',
     'q2doc.external_links',
 ]
 
+
+generate_plugin_rst()
```

Comment:
Can you create a new config section _above_ `# -- General configuration ---` that includes all of the code for this dynamically generated content? That would include the dummy extension registration you hotwired in, belwo.